src/jp2_remediator/main.py

```diff
@@ -37,18 +37,50 @@ def main():
     )
 
     # Subparser for processing all JP2 files in an S3 bucket
-    bucket_parser = subparsers.add_parser(
-        "bucket", help="Process all JP2 files in an S3 bucket"
+    s3_bucket_parser = subparsers.add_parser(
+        "s3", help="Process JP2 files in an S3 bucket"
     )
-    bucket_parser.add_argument(
-        "bucket", help="Name of the AWS S3 bucket to process JP2 files from"
+    s3_bucket_parser.add_argument(
+        "input_bucket", help="Name of the AWS S3 bucket to process JP2 files from"
     )
-    bucket_parser.add_argument(
-        "--prefix", help="Prefix of files in the AWS S3 bucket (optional)",
-        default=""
+    s3_bucket_parser.add_argument(
+        "--input-prefix", help="Prefix of files in the AWS S3 bucket (optional)", default=""
     )
-    bucket_parser.set_defaults(
-        func=lambda args: processor.process_s3_bucket(args.bucket, args.prefix)
+    s3_bucket_parser.add_argument(
+        "--output-bucket", help="Name of the AWS S3 bucket to upload modified files (optional)"
+    )
+    s3_bucket_parser.add_argument(
+        "--output-prefix", help="Prefix for uploaded files in the output bucket (optional)", default=""
+    )
+    s3_bucket_parser.set_defaults(
+        func=lambda args: processor.process_s3_bucket(
+            args.input_bucket, args.input_prefix, args.output_bucket, args.output_prefix
+        )
+    )
+
+    # Subparser for processing a single JP2 file in S3
+    s3_file_parser = subparsers.add_parser(
+        "s3-file", help="Process a single JP2 file in S3"
+    )
+    s3_file_parser.add_argument(
+        "input_bucket", help="Name of the AWS S3 bucket containing the JP2 file"
+    )
+    s3_file_parser.add_argument(
+        "--input-key", help="Key (path) of the JP2 file in the S3 bucket", required=True
+    )
+    s3_file_parser.add_argument(
+        "--output-bucket", help="Name of the AWS S3 bucket to upload the modified file (optional)"
+    )
+    s3_file_parser.add_argument(
```

Comment:
We can remove this argument

Reply:
keep this one, after discussion with andrew